packages/cli/commands/project/migrateApp.js

```diff
@@ -0,0 +1,190 @@
+const path = require('path');
+const {
+  addAccountOptions,
+  addConfigOptions,
+  getAccountId,
+  addUseEnvironmentOptions,
+} = require('../../lib/commonOpts');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { loadAndValidateOptions } = require('../../lib/validation');
+const {
+  createProjectPrompt,
+} = require('../../lib/prompts/createProjectPrompt');
+const { i18n } = require('../../lib/lang');
+const {
+  fetchPublicApp,
+  getMigrationStatus,
+  migratePublicApp,
+  validateAppId,
+} = require('../../lib/publicApps');
+const { poll } = require('../../lib/polling');
+const { uiBetaTag, uiLine, uiLink } = require('../../lib/ui');
+const SpinniesManager = require('../../lib/ui/SpinniesManager');
+const {
+  logApiErrorInstance,
+  ApiErrorContext,
+} = require('../../lib/errorHandlers/apiErrors');
+const { EXIT_CODES } = require('../../lib/enums/exitCodes');
+const { promptUser } = require('../../lib/prompts/promptUtils');
+const { isAppDeveloperAccount } = require('../../lib/accountTypes');
+const { getCwd } = require('@hubspot/local-dev-lib/path');
+const { logger } = require('@hubspot/local-dev-lib/logger');
+const { getAccountConfig } = require('@hubspot/local-dev-lib/config');
+const { downloadProject } = require('@hubspot/local-dev-lib/api/projects');
+const { extractZipArchive } = require('@hubspot/local-dev-lib/archive');
+const { getHubSpotWebsiteOrigin } = require('@hubspot/local-dev-lib/urls');
+const { fetchProject } = require('@hubspot/local-dev-lib/api/projects');
+
+const i18nKey = 'cli.commands.project.subcommands.migrateApp';
+
+exports.command = 'migrate-app';
+exports.describe = uiBetaTag(i18n(`${i18nKey}.describe`), false);
```

Comment:
Will we want to hide this initially?

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1052/commits/62039a6495b1b5eb20d8ddd746508b2daf69a8d4